adobe/react-spectrum
# fix(#4157): useSpinButton: rapidly changing values causes live region to announce values that are no longer relevant

Closes #4157 

## ✅ Pull Request Checklist:

- [x] Included link to corresponding [React Spectrum GitHub Issue 4157](https://github.com/adobe/react-spectrum/issue/4157).

## 📝 Test Instructions:
1. open https://reactspectrum.blob.core.windows.net/reactspectrum/c655ca8987690ae574c5f7dbe3543b59b9e42c93/docs/react-spectrum/NumberField.html#example
2. With a screen reader running, navigate to set focus on the NumberField example with label as "Width" and value 1024.'
3. Press Enter to begin editing the field in forms mode, then press and hold ArrowUp or ArrowDown to increment or decrement the value rapidly by ~20 steps.
4. Screen reader should announce the final value without announcing a bunch of interim values.

## 🧢 Your Project:

Adobe/Accessibility
